CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+## [0.0.18] - 2020-06-03
```

Comment:
You won't know what version this will be until it's released, so when merging to develop you don't want to actually add a release name here

Check https://keepachangelog.com/en/1.1.0/ for details